### DISCUSSION TITLE
🧩 FEATURE REQUEST: Feeds from Only Favourite Sources

### DISCUSSION THREAD

**Discussion by ArchTaqi** [dailydotdev/daily#1008] [Feature Requests]
## Feature Request

Feeds from only my favorite sites. 

### Description

It would be great if we have the feature to have favourite sources i.e InfoQ, Netflix engineering blog etc. So that my feeds only shows up article from this sites.

![Screenshot from 2021-08-28 16-21-15](https://user-images.githubusercontent.com/6747841/131216346-a7cd4715-8d80-4370-a208-0ef5fd1444f9.png)

**Comment by idoshamun** [comment]
Hey @ArchTaqi,

So you want to disable all sources and then select only a few that you like?

**Comment by ArchTaqi** [comment]
@idoshamun Yes.

**Comment by idoshamun** [comment]
We'll discuss it in our upcoming product meeting. Thanks :)
